charts/opentelemetry-collector/examples/deployment-use-existing-configMap/daemonset-values.yaml

```diff
@@ -0,0 +1,24 @@
+mode: daemonset
```

Comment:
You can remove this file, the deployment-values.yaml is sufficient for an example